### PR TITLE
fix(FEC-11288): define product version by core components

### PR DESCRIPTION
**Pull request by Yuvalke**

### CODE REVIEW

**Review by yairans — CHANGES_REQUESTED** [review]

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php [hunk 1]
```diff
@@ -582,16 +585,14 @@ private function setFixVersionsNumber()
 		$isCanaryVersionRequired = array_search(self::CANARY, $this->bundleConfig) !== false;
 
 		$isAllPackagesSameVersion = true;
+		$packageVersion = null;
 
 		if ($isLatestVersionRequired || $isBetaVersionRequired || $isCanaryVersionRequired) {
 
-			list($latestVersionMap, $latestProductVersion) = $this->getConfigByVersion("latest");
+			list($latestVersionMap, $latestProductVersion, $corePackages) = $this->getConfigByVersion("latest");
```
[the remark this answers is not in the document]
Comment (Yuvalke) [reply alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:592]: why? I thought to take the latest only, it's our stable package config.

[the remark this answers is not in the document]
Comment (yairans) [reply alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:592]: don't we report the product version on beta and canary?

[the remark this answers is not in the document]
Comment (yairans) [reply alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:592]: ignore. you are right

**Review by yairans — APPROVED** [review]

**Comment by MosheMaorKaltura** [comment]
reviewed, please make sure change the base to 17.4

**Comment by yairans** [comment]
@Yuvalke change the base to Quasar-17.5.0

**Pull request merged**